### DISCUSSION TITLE
✅Key pressed is not working.

### DISCUSSION THREAD

**Discussion by ajaysimha** [isontheline/pro.webssh.net#1251] [Q&A] [answered]

**Comment by ajaysimha** [comment, marked as answer]
I was totally stumped to understand that on ipads holding keys down simply does not repeat the keys.
New one for me. Not a webssh issue

**Reply by isontheline** [reply]
You’re right @ajaysimha hope iOS will allow this behavior in a future release 

I wish you a nice day ☀️
